include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -37,212 +37,29 @@ namespace seqan3::detail
-        for (auto f : positional_option_calls)
-            f();
-
-        // add options and flags if specified
-        if (!parser_set_up_calls.empty())
-            print_section("Options");
-
-        // each call will evaluate the function print_list_item()
-        for (auto f : parser_set_up_calls)
-            f();
-
-        if (!meta.examples.empty())
-        {
-            print_section("Examples");
-            for (auto example : meta.examples)
-                print_line(example);
-        }
-
-        print_footer();
-
-        std::exit(EXIT_SUCCESS); // program should not continue from here
-    }
-
-    /*!\brief Adds a print_section call to parser_set_up_calls.
-     * \param[in] title The title of the section of the help page.
-     */
-    void add_section(std::string const & title)
-    {
-        parser_set_up_calls.push_back([this, title] ()
-        {
-            print_section(title);
-        });
-    }
-
-    /*!\brief Adds a print_subsection call to parser_set_up_calls.
-     * \param[in] title The title of the subsection of the help page.
-     */
-    void add_subsection(std::string const & title)
-    {
-        parser_set_up_calls.push_back([this, title] ()
-        {
-            print_subsection(title);
-        });
-    }
-
-    /*!\brief Adds a print_line call to parser_set_up_calls.
-     * \param[in] text The line text to be printed to the help page.
-     * \param[in] line_is_paragraph True if you want a new line at the end.
-     */
-    void add_line(std::string const & text, bool line_is_paragraph)
-    {
-        parser_set_up_calls.push_back([this, text, line_is_paragraph] ()
-        {
-            print_line(text, line_is_paragraph);
-        });
-    }
-
-    /*!\brief Adds a seqan3::print_list_item call to parser_set_up_calls.
-     * \param[in] key The key of the key-value pair list item.
-     * \param[in] desc The key of the key-value pair list item.
-     */
-    void add_list_item(std::string const & key, std::string const & desc)
-    {
-        parser_set_up_calls.push_back([this, key, desc] ()
-        {
-            print_list_item(key, desc);
-        });
-    }
-
-    /*!\brief Stores all meta information about the application
-     *
-     * \details
-     *
-     * This needs to be a member of format_parse, because it needs to present
-     * (not filled) when the parser_set_up_calls vector is filled, since all
-     * printing functions need some meta information.
-     * The member variable itself is filled when copied over from the argument_parser
-     * when calling format_parse::parse. That way all the information needed are
-     * there, when the actual printing starts.
-     *
-     * This function is not private because it is needed for short but nicely
-     * formatted (error) output to the command line.
-     */
-    argument_parser_meta_data meta;
+    //!\copydoc format_help_base(bool)
+    format_help(bool const advanced) : base_type{advanced}
```

Comment:
it looks like this constructor is not in the doxygen group for constructors anymore? 🤔